sdk/go/common/apitype/updates.go

```diff
@@ -70,6 +70,22 @@ type UpdateMetadata struct {
 	Environment map[string]string `json:"environment"`
 }
 
+type MessageSeverity string
+
+const (
+	SeverityWarning MessageSeverity = "warning"
+	SeverityError   MessageSeverity = "error"
+	SeverityInfo    MessageSeverity = "info"
```

Comment:
Minor: Typically if you're adding a shared prefix or suffix to the constants, it should match the type name. So these should either be `MessageSeverity{Warning,Error,Info}`, or the type should be `type Severity`.

I'll defer to you on the choice between the two, but I would personally opt for Severity.